src/Messenger/IndexationRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace JoliCode\Elastically\Messenger;
+
+final class IndexationRequest
+{
+    private $op;
+    private $type;
+    private $id;
+
+    public function __construct(string $type, string $id, string $op = IndexationRequestHandler::OP_INDEX)
+    {
+        if (!in_array($op, IndexationRequestHandler::OPS, true)) {
```

Comment:
Could you run php-cs-fixer to automatically add `\` in front of all PHP functions that are opitmised by opcache

Reply:
To be done in another PR, I already runned `make cs` but adding `\` is not part of our configuration.